### ISSUE TITLE
Create a simple QA NN

### ISSUE THREAD

**Issue by David-Ciz** [David-Ciz/silver-truth#17]
- Install and try pytorch lightning hello world
- efficient net, resnet, custom architekture as models
- Inputs changed based on the model input requirements
- pretrained weights vs random initialization
- regression output layer (loss functions based on this info)
- Filter out and denote train and val data based on the parquet
- inputs are cell images - two layers as inputs, shape 64x64x2 (size exampled)
- outputs are jaccard scores
- find out good metrics, come up with your own
- Train for 500 epochs
- evaluate on val dataset
- evaluate on test dataset
